message MSG_DM_DC_DOMAIN_SYSINFO_GET
{
    optional string timestamp = 1;
}

message MSG_DC_DM_DOMAIN_SYSINFO_GET_ACK
{
    optional int32 statusCode = 1;
    message CS_SYS_INFO
    {
        optional string nodeStatus = 1;
        optional uint32 core = 2;
        optional uint32 memSize = 3;
    }
    repeated CS_SYS_INFO singleNodeSysInfo = 2;
    optional string timeStamp = 3;
}